test/tests/functional/pbs_basil_support.py

```diff
@@ -308,3 +309,42 @@ def test_cray_login_node(self):
                              ("%s\t: pbs node has %s=%s but login "
                               "node has %s=%s") %
                              (mom_id, rsc, val, rsc, rval))
+
+    def test_hbmemm_rsc(self):
+        """
+        Create a job that requests enough HBMEM. Submit the job to
+        the Server. Check if the job is in the 'R' state and if the
+        job runs on a KNL vnode. Delete the job.
+        """
+
+        knl_vnodes = self.get_knl_vnodes()
+
+        if len(knl_vnodes) == 0:
+            self.skipTest(reason='No KNL vnodes present')
+            return
+        else:
+            self.logger.info("KNL vnode list: %s" % (knl_vnodes))
+
+        # Add 'hbmem' to the resources line in sched_config.
+        self.scheduler.add_resource('hbmem')
```

Comment:
This step should not be necessary since interface 6 of your design says this will already be set.  Is there a reason you are setting it here?

Reply:
This we need for simulator.